charmcraft.yaml

```diff
@@ -11,4 +11,5 @@ bases:
       channel: "22.04"
 parts:
   charm:
+    build-packages: [git, rustc, cargo]
```

Comment:
Should we add a comment indicating which components needed these deps to build?

Reply:
I don't know what components needed these packages. I just realized that I needed it to build.